src/sota_tools/garage_deploy.cc

```diff
@@ -122,6 +122,14 @@ int main(int argc, char **argv) {
     LOG_FATAL << e.what();
     return EXIT_FAILURE;
   }
+
```

Comment:
Would be nice to have a check for dry run, like in the rest of the code

Reply:
Yeah, that makes sense! Thanks.

Reply:
I haven't tested the code yet, just pushed it here to get your advices. Am I on the right track?

Reply:
Yes, it's going in the right direction. I would just move `CheckRefValid` in a new source file.